### PR TITLE
HUB-488 Content updates to Self Service

### PR DESCRIPTION
**Pull request by j-maynard**
We continue to improve the Self Service tool and user journeys through
the app.  These minor changes we hope will make the self service tool
easier to user.

Jira Ticket can be found at: https://govukverify.atlassian.net/secure/RapidBoard.jspa?rapidView=132&projectKey=HUB&modal=detail&selectedIssue=HUB-488

Request change doc can be found at: https://docs.google.com/document/d/112KYBzbOCOI2js0IeJKHCe2QEZ7eGB961qg1ce9I3GI/edit#

### First change requested
**From:**
> You need to manage your certificates to make sure your service stays connected to GOV.UK Verify.
>If your connection has been broken or compromised, read [this guidance].

**To:**
> Manage your certificates to make sure your service stays connected to GOV.UK Verify. 
>When your certificates are expiring, you can rotate them in any order. You can also rotate more than one certificate at the same time.
>You should follow the [instructions in the documentation], which also explains what to do [if your connection has been broken or compromised].

Screen shot:
Screenshot_20200305_030809

### Second change requested
**From:**
>[Rotate more certificates]

**To:**
>| You can [rotate more certificates] while you wait for the email.
*With the following note:*
Move this content so it’s above the ‘Next steps’. Put it into a callout, to show that it’s slightly outside of the process being described on the page.

Screen shot:
image

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -367,9 +367,14 @@ en:
     wait_for_an_email: Wait for an email from GOV.UK Verify confirming your new signing certificate is in use
     how_long_it_takes: This usually takes a few minutes from the time you uploaded the new signing certificate.
     delete_old_signing_key_and_cert: Then you can delete the old signing key and certificate from your %{component} configuration.
-    maintain_connection: You need to manage your certificates to make sure your service stays connected to GOV.UK Verify.
-    connection_broken_or_compromised_guidance: If your connection has been broken or compromised, read
-    connection_broken_or_compromised_link: this guidance
+    # maintain_connection: You need to manage your certificates to make sure your service stays connected to GOV.UK Verify.
```
Comment (AmritSidhu) [inline config/locales/en.yml]: Need to uncomment these?

Comment (j-maynard) [reply config/locales/en.yml]: Opps... Forgot those... Take another look, I've amended the last commit.

File: app/views/user_journey/confirmation.html.erb [hunk 2]
```diff
@@ -3,6 +3,10 @@
 <%= render 'shared/certificate_details', certificate: @certificate %>
 
 <h1 class="govuk-heading-l"><%= t 'user_journey.adding_certificate_to_config' %></h1>
+  
+<div class="govuk-inset-text">
```
Comment (galund) [inline app/views/user_journey/confirmation.html.erb]: Why not `<p>`?

Comment (AmritSidhu) [reply app/views/user_journey/confirmation.html.erb]: Its whats in the design system guide as the example 🤷🏽‍♂️

**Comment by galund** [comment]
So this is complicated to explain, but those translation strings need to be postfixed `_html`, because you've implicitly made them need to be markup (rather than plain text) by doing the substitution in the way you have. (There might be other ways of fixing this – I don't think the way we do translations here is really correct – but this seems the simplest fix to make clear what's going on?)

**Comment by AmritSidhu** [comment]
I think branch is failing due to tests relating to @jkunle work on CertificateInUseEvent. The array being used to compare is returning the items in a different order. Will talk to Ola about this

File: config/locales/en.yml [hunk 3]
```diff
@@ -387,6 +389,7 @@ en:
       SP:
         encryption: https://www.docs.verify.service.gov.uk/rotating-your-keys-and-certificates/service-provider-encryption/#rotate-your-service-provider-encryption-key-and-certificate
         signing: https://www.docs.verify.service.gov.uk/rotating-your-keys-and-certificates/service-provider-signing/#rotate-your-service-provider-signing-key-and-certificate
+      instructions: https://www.docs.verify.service.gov.uk/rotating-your-keys-and-certificates/#rotating-your-keys-and-certificates
```
Comment (jakubmiarka) [inline config/locales/en.yml]: this is the same as two lines below, `rotation_documentation`, so maybe used that instead?

**Comment by bethwalton** [comment]
Hey Jamie, on the second change, can the line in the callout ("You can [rotate more certificates] while you wait for the email") come after the first sentence on that page? 

So it goes like this:

This usually takes a few minutes. We'll email you when your new [certificate] is in use.
         
        | You can [rotate more certificates] while you wait for the email 

Next steps 
etc 

Thanks :)

File: config/locales/en.yml [hunk 4]
```diff
@@ -461,12 +464,13 @@ en:
...
       publish_failed_title: Publish failed
       what_to_expect: This usually takes a few minutes. We'll email you when your new %{type} certificate is in use.
+      rotate_more: You can %{rotate_more_link} while you wait for the email.
       next_steps: Next steps
       rotate_other_certificates: You can rotate other certificates straight away - you do not need to wait for the email.
       received_email: Once you've received the email, delete the old %{usage} key and certificate from your %{component} configuration.
       received_email_to_replace: Once you've received the email, replace the old %{usage} key with the new one in your %{component} configuration.
       received_email_to_promote: Once you've received the email, go to your %{component} configuration. Move your 'secondary' %{usage} key and certificate to the 'primary' position, and delete the old key and certificate.
-      rotate_more: Rotate more certificates
+      rotate_more_link: rotate more certificates
```
Comment (AmritSidhu) [inline config/locales/en.yml]: Where you have made this lower case its causing the tests fail

File: pgadmin-env.list [hunk 5]
```diff
@@ -0,0 +1,3 @@
+PGADMIN_DEFAULT_EMAIL=jamie.maynard@me.com
+PGADMIN_DEFAULT_PASSWORD=8lScP10eY1L3oeXbBM1E
```
Comment (jakubmiarka) [inline pgadmin-env.list]: oh no! 😱

File: pgadmin-env.list [hunk 6]
```diff
@@ -0,0 +1,3 @@
+PGADMIN_DEFAULT_EMAIL=jamie.maynard@me.com
```
Comment (jkunle) [inline pgadmin-env.list]: I think you checked this in by mistake

Comment (j-maynard) [reply pgadmin-env.list]: Removed... Sorry about that... My Docker Postgres script needs tweaking.

File: pgadmin-env.list (hunk 5 above)
Comment (j-maynard) [reply pgadmin-env.list]: Reset back to the Master version... my local environment is using a Docker version of Postgres.

**Comment by bethwalton** [comment]
Content looks great :) Thank you!

File: config/locales/en.yml [hunk 7]
```diff
@@ -367,9 +367,11 @@ en:
     wait_for_an_email: Wait for an email from GOV.UK Verify confirming your new signing certificate is in use
     how_long_it_takes: This usually takes a few minutes from the time you uploaded the new signing certificate.
     delete_old_signing_key_and_cert: Then you can delete the old signing key and certificate from your %{component} configuration.
-    maintain_connection: You need to manage your certificates to make sure your service stays connected to GOV.UK Verify.
-    connection_broken_or_compromised_guidance: If your connection has been broken or compromised, read
-    connection_broken_or_compromised_link: this guidance
+    maintain_connection_p1: Manage your certificates to make sure your service stays connected to GOV.UK Verify. 
+    maintain_connection_p2: When your certificates are expiring, you can rotate them in any order. You can also rotate more than one certificate at the same time.
```
Comment (galund) [inline config/locales/en.yml]: I'd have preferred these combined as one html block, but not gonna fuss about that now

File: spec/system/visit_user_journey_confirmation_page_spec.rb [hunk 8]
```diff
@@ -18,7 +18,7 @@
       visit confirmation_path(event.component.encryption_certificate.id)
       expect(page).to have_content COMPONENT_TYPE::MSA_SHORT
       expect(page).to have_content 'delete the old encryption key and certificate from your MSA configuration'
-      click_link 'Rotate more certificates'
+      click_link t('user_journey.confirmation.rotate_more_link')
```
Comment (galund) [inline spec/system/visit_user_journey_confirmation_page_spec.rb]: I like this DRY-ness :clap:

**Pull request merged**